assets/js/atomic/blocks/product-elements/rating/support.ts

```diff
@@ -20,7 +17,7 @@ export const supports = {
 			fontSize: true,
 			__experimentalSkipSerialization: true,
 		},
-		...( hasSpacingStyleSupport() && {
+		...( __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
The `tyepeof` keyword is missing:

```suggestion
		...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```